planetiler-core/src/main/java/com/onthegomap/planetiler/reader/osm/OsmReader.java

```diff
@@ -293,7 +293,8 @@ void processPass1Blocks(Iterable<? extends Iterable<? extends OsmElement>> block
   }
 
   private static boolean isMultipolygon(OsmElement.Relation relation) {
-    return relation.hasTag("type", "multipolygon", "boundary", "land_area");
+    return relation.hasTag("type", "multipolygon", "boundary", "land_area") &&
+      relation.members().stream().anyMatch(m -> m.type() == OsmElement.Type.WAY && "outer".equals(m.role()));
```

Comment:
Planetiler uses logic ported from imposm3 which ignores the "outer" / "inner" tags when reconstructing the polygon and instead infers it by analyzing the geometries to see which ones contain the others so it can be robust to outer/inner being incorrectly specified. So technically planetiler could reconstruct a valid polygon if the multipolygon just contains one or two ways without any outer/inner tags. 

So we could simplify to something like:

```suggestion
      relation.members().stream().anyMatch(m -> m.type() == OsmElement.Type.WAY);
```

what do you think?